Exercise_B/index.html

```diff
@@ -19,16 +19,46 @@
 
   <body>
     <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
+    <div class="container-fluid">
+      <img class="img" src="../Exercise_B/assets/images/logo.png" alt="Navbaricon">
+    </div>
 
+    <!--Banner Jumbotron of the image-->
+    
+        <div class="banner">
+          <div class="container-fluid text-center">
+            <img class="banner" src="../Exercise_B/assets/images/flowershop.jpg" alt="banner">
+            <h1 class ="heading text light">Beautiful Flowers</h1>
+          <button type ="submit" class="btn btn-success" id="learn-more">Learn more
+          </button>
+          </div>
+          
     <!--Four containers with img / text-->
     <!-- 4 containers -->
-
+    <div id="bottom" class="d-flex flex-wrap row text-center px-5">
+      <div class="col-6 d-flex align-items-center justify-content-center px-5 py-3">
+        <img class="h-50" src="../Exercise_B/assets/images/flower-icon.png" atl="Flowers">
+          <p class="card-text p-3">Flowers for every kind of situation</p>
+      </div>
     <!-- Payments Div -->
+    <div class="col-6 d-flex align-items-center justify-content-center px-5 py-3">
+      <img class="h-50" src="../Exercise_B/assets/images/payment-icon.png" alt="Payment">
+        <p class="card-text p-3">All kinds of payments accepted</p>
+    </div>
     <!-- Consultation Div -->
-
+    <div class="col-6 d-flex align-items-center justify-content-center px-5 py-3">
+        <img class="h-50" src="../Exercise_B/assets/images/phone-icon.png" alt="phone">
+          <p class="card-text p-3">Free consultation</p>
+      </div>
     <!-- Delivery Div -->
-
+    <div class="col-6 d-flex align-items-center justify-content-center px-5 py-3">
+        <img class="h-50" src="../Exercise_B/assets/images/truck-icon.png" alt="truck">
+          <p class="card-text p-3">City wide Delivery</p>
+      </div>
     <!--Scripts required for bootstrap CDN-->
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.3/dist/umd/popper.min.js" integrity="sha384-eMNCOe7tC1doHpGoWe/6oMVemdAVTMs2xqW4mwXrXsW0L84Iytr2wi5v2QjrP/xp" crossorigin="anonymous"></script>
+  
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.min.js" integrity="sha384-cn7l7gDp0eyniUwwAZgrzD06kc/tftFf19TOAs2zVinnD/C7E91j9yyk5//jjpt/" crossorigin="anonymous"></script>
+  
   </body>
 </html>

```

Comment:
For this, there could have been a better implentation when working with grid. For using the column, system that could have been implemented by using sizing such as xxl, xl, l and other sizes. Additionally, there has to be a background behind which is green. I would suggest, looking at the answer. Otherwise i can explain this to you later on. Great work htought.